selfdrive/camerad/cameras/camera_qcom.cc

```diff
@@ -2039,10 +2047,13 @@ static void* ops_thread(void* arg) {
   zsock_t *sensor_sock = zsock_new_sub(">tcp://127.0.0.1:8003", "");
   assert(sensor_sock);
 
+  zsock_t *controls_sock = zsock_new_sub(">tcp://127.0.0.1:8007", "");
+  assert(controls_sock);
+
   zsock_t *terminate = zsock_new_sub(">inproc://terminate", "");
   assert(terminate);
 
-  zpoller_t *poller = zpoller_new(cameraops, sensor_sock, terminate, NULL);
+  zpoller_t *poller = zpoller_new(cameraops, sensor_sock, controls_sock, terminate, NULL);
```

Comment:
Need to use a msgq sock for controlsState

Reply:
lol. fixed